datasetinsights/io/gcs.py

```diff
@@ -1,11 +1,19 @@
 import logging
 import os
+from os import makedirs
+from os.path import basename, isdir
 from pathlib import Path
 
 from google.cloud.storage import Client
 
+from datasetinsights.io.download import validate_checksum
+from datasetinsights.io.exceptions import ChecksumError, DownloadError
+
 logger = logging.getLogger(__name__)
 
+MD5 = "MD5"
+REPAD = "=="
+
 
 class GCSClient:
```

Comment:
@adason Why are the functions below not a part of the GCSClient?
gcs_bucket_and_path, copy_folder_to_gcs, download_file_from_gcs